crates/forge_loader/src/manifest.rs

```diff
@@ -249,10 +250,55 @@ struct AssetsImportType<'a> {
     import_status: JustFunc<'a>,
 }
 
+// Rovo Modules
+#[derive(Default, Debug, Clone, PartialEq, Eq, Deserialize)]
+pub struct RovoAgent<'a> {
+    pub key: &'a str,
+    pub name: &'a str,
+    pub description: Option<&'a str>,
+    pub icon: Option<&'a str>,
+    pub prompt: String, // as may be multiline
+    #[serde(default, rename = "conversationStarters", borrow)]
+    pub conversation_starters: Vec<&'a str>,
+    #[serde(default, borrow)]
+    pub actions: Vec<&'a str>,
+    #[serde(rename = "followUpPrompt")]
+    pub follow_up_prompt: Option<&'a str>,
+}
+
+#[derive(Debug, Clone, PartialEq, Eq, Deserialize)]
+pub struct ActionInput<'a> {
+    pub title: &'a str,
+    #[serde(rename = "type")]
+    pub input_type: &'a str,
+    pub required: bool,
+    pub description: Option<&'a str>,
+}
+
+#[derive(Debug, Clone, PartialEq, Eq, Deserialize)]
+pub struct Action<'a> {
+    pub key: &'a str,
+
+    // TODO: Rovo Action documentation states this is mandatory field but example apps don't use it, a typo? or are "required" fields not strictly enforced?
+    // eg: https://bitbucket.org/atlassian/rovo-data-discoverability/src/2743f6ca42589c7d1fb2d50e0d9e65e5ef1aebbe/manifest.yml#lines-3
+    pub name: &'a str,
```

Comment:
If it's not mandatory or you don't know, you can just leave it off, since we really only care about `function`.